src/main/kotlin/com/autonomousapps/subplugin/ProjectPlugin.kt

```diff
@@ -835,6 +840,8 @@ internal class ProjectPlugin(private val project: Project) {
     )
   }
 
+  private fun Project.buildName(): String = serviceOf<BuildState>().buildIdentifier.name
```

Comment:
how safe / future-proof is this?

and since it's called multiple times, is there benefit in caching the value after first lookup?

Reply:
Very good point. I asked in the Gradle Community Slack and Stefan pointed me at a better solution. This is now updated.

I also discovered that with Gradle 8.2, `buildName` is replaced by `buildPath`. We should do a version check and use that for new Gradle versions. I added a FIXME to do this, once the plugin build has been updated to the latest Gradle version.

See: https://docs.gradle.org/8.2-rc-2/userguide/upgrading_version_8.html#build_identifier_name_and_current_deprecation